extension/chrome/elements/compose-modules/formatters/encrypted-mail-msg-formatter.ts

```diff
@@ -63,7 +63,8 @@ export class EncryptedMsgMailFormatter extends BaseMailFormatter {
     const { data: pubEncryptedNoAtts } = await this.encryptDataArmor(Buf.fromUtfStr(pgpMimeNoAtts), undefined, pubs, signingPrv); // encrypted only for pubs
     const atts = this.createPgpMimeAtts(pubEncryptedNoAtts).concat(await this.view.attsModule.attach.collectEncryptAtts(pubs)); // encrypted only for pubs
     const emailIntroAndLinkBody = await this.formatPwdEncryptedMsgBodyLink(short);
-    return await SendableMsg.create(this.acctEmail, { ...this.headers(newMsg), body: emailIntroAndLinkBody, atts, isDraft: this.isDraft });
+    //return await SendableMsg.create(this.acctEmail, { ...this.headers(newMsg), body: emailIntroAndLinkBody, atts, isDraft: this.isDraft });
+    return await SendableMsg.createOpenPGP(this.acctEmail, { ...this.headers(newMsg), body: emailIntroAndLinkBody, atts, isDraft: this.isDraft });
```

Comment:
I think this one is a bit of a special case - a password-protected message that doesn't necessarily follow any known PGP format - it's a mix. Let's call it `createPwdMsg`, and then later I'll review how similar it is to the other method, in case we decide to merge it with one of them.

Reply:
ok